### PR TITLE
edited strings/anagram.py

### PR DESCRIPTION
**Pull request by Leoriem-code**
### Describe your change:

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [x] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by Leoriem-code** [comment]
the build can't complete because the file `anagrams.py` use `words.py`, file that is not accessible during build/test

**Comment by cclauss** [comment]
Take a look at https://github.com/TheAlgorithms/Python/blob/cecf43d6481173e831af829da77911e1a3868a6c/project_euler/problem_102/sol1.py#L65

**Comment by Leoriem-code** [comment]
Thanks, let's hope it works

File: strings/anagrams.py [hunk 1]
```diff
@@ -1,35 +1,44 @@
...
+    >>> anagram('final')
+    ['final']
+    """
+    return word_bysig[signature(my_word)]
 
-def signature(word):
-    return "".join(sorted(word))
 
+data: str = Path(__file__).parent.joinpath("words.txt").read_text(encoding="utf-8")
+word_list = sorted(list({word.strip().lower() for word in data.split("\n")}))
```
Comment (cclauss) [inline strings/anagrams.py:33]:
```suggestion
word_list = sorted({word.strip().lower() for word in data.splitlines()})
```

**Review by cclauss — APPROVED** [review]
We could have used `pathlib` for both reading and writing but let's land this one.

**Pull request merged**